models/social_connection.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
Why have this as a join model instead of a dict property on `Team` objects? I'm not sure which approach would ultimately be cleaner.